cob_openni2_tracker/launch/body_tracker_nodelet.launch

```diff
@@ -21,7 +21,7 @@
 
 
 	<node pkg="nodelet" type="nodelet" name="body_tracker" args="load cob_openni2_tracker/body_tracker_nodelet /$(arg camera_name)/$(arg camera_name)_nodelet_manager" output="screen" cwd="ROS_HOME">
-		<param name="camera_frame_id" type="str" value="$(arg camera_name)_link" /> <!-- $(arg camera_name)_depth_optical_frame -->
+		<param name="camera_frame_id" type="str" value="$(arg camera_name)_rgb_frame" />   <!-- "$(arg camera_name)_link" -->
```

Comment:
Hehe yes, you are right. That came from my branch, but it is not yet correct. Anyways, I will accept your pull request and then upload my correction to the coordinate system definitions. Please update your software after one hour with my current status. By the way, are you using a Kinect or Asus device? I assume that there is a difference in coordinate systems between Kinect and Asus, but I do not have a Kinect at hand to test this. So maybe we now have wrong coordinate systems with a Kinect after my update.